src/parameter_input.cpp

```diff
@@ -199,6 +205,15 @@ void ParameterInput::LoadFromStream(std::istream &is) {
 
     if (!continuing) {
       if (param_name != "") {
+        if (DoesParameterExist(block_name, param_name)) {
+          msg << "### FATAL ERROR in function [ParameterInput::LoadFromStream]"
+              << std::endl
+              << "Block/parameter " << block_name << "/" << param_name
+              << " appears more than once in input" << std::endl
+              << "NOTE: input block and parameter names are NOT case sensitive"
+              << std::endl;
```

Comment:
`PARTHENON_THROW` takes care of the function location and final line break. (It spits out a file and line number.) I think the redundant information can't hurt, so this is fine as is. Just a heads up.